mflowgen/tile_array/Tile_PE/get_Tile_PE_outputs.sh

```diff
@@ -8,6 +8,8 @@ else
     make cadence-pegasus-lvs
 fi
 
+make pwr-aware-gls
+
```

Comment:
same

Reply:
@alexcarsello I do agree, but since this is the only simulation whose function is based on physical design, I say we keep it. You really shouldn't proceed to build the tile_array if the tiles can't properly toggle on and off IMO. If we had this in, we would have discovered the bug much earlier.

Reply:
Furthermore, this means any time we want to do CI we need to take the tiles all the way through PnR

Reply:
You could say the same about having it in our CI tests, but sure. Not a huge deal.